src/ui/router/Browse.js

```diff
@@ -119,18 +118,21 @@ define([
         }
 
         function navigateToFirstChildOfRoot() {
-            openmct.objects.get('ROOT').then(rootObject => {
-                openmct.composition.get(rootObject).load()
-                    .then(children => {
-                        let lastChild = children[children.length - 1];
-                        if (!lastChild) {
-                            console.error('Unable to navigate to anything. No root objects found.');
-                        } else {
-                            let lastChildId = openmct.objects.makeKeyString(lastChild.identifier);
-                            openmct.router.setPath(`#/browse/${lastChildId}`);
-                        }
-                    });
-            });
+            openmct.objects.get('ROOT')
+                .then(rootObject => {
+                    openmct.composition.get(rootObject).load()
+                        .then(children => {
+                            let lastChild = children[children.length - 1];
+                            if (!lastChild) {
+                                console.error('Unable to navigate to anything. No root objects found.');
+                            } else {
+                                let lastChildId = openmct.objects.makeKeyString(lastChild.identifier);
+                                openmct.router.setPath(`#/browse/${lastChildId}`);
+                            }
+                        })
+                        .catch(e => console.error(e));
```

Comment:
What errors were you seeing here?

Reply:
for `npm run test`
```
An error was thrown in afterAll
  Unhandled promise rejection: TypeError: openmct.composition.get(...) is undefined in indexTest.js line 22608 > eval (line 1)
  navigateToFirstChildOfRoot/<@webpack-internal:///./src/ui/router/Browse.js:1:13382
```
check full error here -> https://app.circleci.com/pipelines/github/nasa/openmct/3315/workflows/0fc7ad2c-bbce-40ee-80fc-c0be4bd1538a/jobs/12171/parallel-runs/0/steps/0-106